lua/telescope/utils.lua

```diff
@@ -192,6 +193,11 @@ function utils.display_termcodes(str)
   return str:gsub(string.char(9), "<TAB>"):gsub("", "<C-F>"):gsub(" ", "<Space>")
 end
 
+function utils.get_job_command_output(cmd)
+  local command = table.remove(cmd, 1)
+  return Job:new({ command = command, args = cmd }):sync()
+end
```

Comment:
I wanna use plenary job over `io.popen` for some time now.
But we kinda need to keep `io.popen` around of man page and someone wanted to do a pipe here to fix issues with `mac os`. See #413